js/main.js

```diff
@@ -143,12 +156,95 @@ function round(n) {
     return number;
 }
 
-function saveBallValues(left, bottom, angle){
+function saveBallValues(left, bottom, angle) {
     ball.position.left = left;
     ball.position.bottom = bottom;
     ball.position.angle = angle;
 }
 
+window.addEventListener("keydown", function (event) {
+    keysDown[event.keyCode] = true;
+}, false);
+window.addEventListener("keyup", function (event) {
+    delete keysDown[event.keyCode];
+}, false);
+
+function checkPressedKeys() {
```

Comment:
Warum prüft ihr denn den KeyPress in einer separation funktion? Warum nicht einfach direkt in der onKeyDown funktion?